source/ui/SpriteAnimation.js

```diff
@@ -303,6 +349,8 @@
 		stop: function () {
 			this.$.spriteImage.applyStyle('-webkit-animation-name', null);
```

Comment:
Should we conditionalize these statements on `this.useCssAnimation` as well, to correspond to `start`?